src/breedsRouter.js

```diff
@@ -1,16 +1,26 @@
+// TODO provide DEL, POST, PUT routes also. 
 
 const express = require('express');  
 const breedsRouter = express.Router();
 
-const BreedRepository = require('./breedsDb.js');
+const breedRepository = require('./breedsDb.js');
+
+
+const isInvalidString = (value) => {
```

Comment:
These seem familiar! could you consider extracting them into a separate file from catsRouter/breedsRouter to avoid duplication? You could also consider unit tests for the individual functions if they're not covered by tests already, as it should be pretty straightforward to test these in isolation from other components.